src/components/course/data/utils.jsx

```diff
@@ -22,12 +22,7 @@ import CreditSvgIcon from '../../../assets/icons/credit.svg';
 import { PROGRAM_TYPE_MAP } from '../../program/data/constants';
 import { programIsMicroMasters, programIsProfessionalCertificate } from '../../program/data/utils';
 import { hasValidStartExpirationDates } from '../../../utils/common';
-
-export function hasCourseStarted(start) {
-  const today = new Date();
-  const startDate = new Date(start);
-  return startDate && today >= startDate;
-}
+import { compareRedeemableOffers, hasCourseStarted as hasCourseStartedTs} from './utils.tsx';
```

Comment:
Is it actually necessary to add the extensions now?  I wonder if we could tweak the webpack config in frontend-build to avoid this. I know I have a (personal) TS project where I don't have to include extensions on my imports. 🤔

Reply:
If we don't have to disambiguate between TypeScript and a Javascript file with identical file paths, then we don't need to include a filename extension.